releasenotes/notes/disable-default-plot-table-330705e3118fac6a.yaml

```diff
@@ -0,0 +1,6 @@
+---
+deprecations:
+  - |
+    Schedule plotting with :py:class:meth:`qiskit.pulse.Schedule.draw` will no
```

Comment:
looks like `:py:class:meth:` causes a warning at docs build. perhaps [`classmethod`](https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#directive-py-classmethod)?

Reply:
Hopefully is fixed now.